src/main/java/games/strategy/engine/framework/startup/ui/PlayerSelectorRow.java

```diff
@@ -23,27 +23,31 @@
 
 class PlayerSelectorRow {
 
+  private static final String PLAYER_TYPE_AI = "AI";
+  private static final String PLAYER_TYPE_DOES_NOTHING = "DoesNothing";
```

Comment:
Yeah, I thought the same thing but decided to leave them just in this class for now.